pages/user.py

```diff
@@ -40,10 +40,10 @@ class BaseUserSettingsPage(OSFBasePage):
 class ProfileInformationPage(BaseUserSettingsPage):
     url = settings.OSF_HOME + '/settings/'
 
-    identity = Locator(By.CSS_SELECTOR, '#profileSettings')
+    identity = Locator(By.CSS_SELECTOR, 'div[id="profileSettings"]')
     middle_name_input = Locator(By.CSS_SELECTOR, '#names > div > form > div:nth-child(5) > input')
     save_button = Locator(By.CSS_SELECTOR, '#names > div > form > div.p-t-lg.p-b-lg > button.btn.btn-success')
-    update_success = Locator(By.CSS_SELECTOR, '.text-success')
+    # update_success = Locator(By.CSS_SELECTOR, '.text-success')
```

Comment:
Commenting this line out will probably break test_user.py line
```profile_settings_page.update_success.here_then_gone()```
So we'll need to find a way to keep it in. Hopefully, the navbars test is only checking the identity of the page it lands on, and isn't checking every locator on the page.

Reply:
Update: we tested this live.  Works if uncommented out, yay!